app/controllers/admin/merchants_controller.rb

```diff
@@ -2,6 +2,7 @@ class Admin::MerchantsController < ApplicationController
   def index
```

Comment:
We should look at a refactor later, it looks like to me you can probably just pass through Merchants and then use your built-in methods in the view to narrow. This works either way, but think it would be neat to get this to one variable passed, which I think you can!